platforms/springboot/src/main/java/io/hawt/springboot/HawtioPlugin.java

```diff
@@ -26,12 +27,31 @@ public class HawtioPlugin {
     private Boolean bustRemoteEntryCache;
     private String pluginEntry;
 
+    /**
+     *
+     * @param url The URL of the remote entry for the plugin. This must be the same location as the Hawtio console.
+     * @param scope The name of the container defined at Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     * @param module The path exposed from Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     */
     public HawtioPlugin(String url, String scope, String module) {
         this.url = url;
         this.scope = scope;
         this.module = module;
     }
 
+    /**
+     *
+     * @param scope The name of the container defined at Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     * @param module The path exposed from Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     *
+     * This constructor automatically uses hawtio address.
```

Comment:
It's not commont to write the comment body at the bottom. Please move it to the top.

Also, it's not precisely "hawtio address" but the address of the application.